### ISSUE TITLE
Unable to create session.

### ISSUE THREAD

**Issue by qazsedcftgbhujmkol** [DIGITALCRIMINAL/OnlyFans#363]
I downloaded via git clone 2 minutes ago.
I ran pip install -r requirements.txt
I ran start_ofd.py
I then opened onlyfans and the config file and added the auth_id, sess, user-agent and auth_uniq_ values. (this took < 30 seconds)
I pressed enter to get the script to load the config file.
I then pressed 0 for onlyfans
I got:
`Unable to create session`
`There's nothing to scrape.`

I tried pressing 0 again twice, just in case.
I get the same thing.

Command line dump:
```
T:\#of2>pip install -r requirements.txt
Collecting requests
  Using cached requests-2.24.0-py2.py3-none-any.whl (61 kB)
Collecting beautifulsoup4
  Using cached beautifulsoup4-4.9.1-py3-none-any.whl (115 kB)
Collecting urllib3
  Using cached urllib3-1.25.10-py2.py3-none-any.whl (127 kB)
Collecting win32-setctime
  Using cached win32_setctime-1.0.1-py3-none-any.whl (3.4 kB)
Collecting lxml
  Using cached lxml-4.5.2-cp38-cp38-win_amd64.whl (3.5 MB)
Collecting pysocks
  Using cached PySocks-1.7.1-py3-none-any.whl (16 kB)
Requirement already satisfied: certifi>=2017.4.17 in c:\python38\lib\site-packages (from requests->-r requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: chardet<4,>=3.0.2 in c:\python38\lib\site-packages (from requests->-r requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: idna<3,>=2.5 in c:\python38\lib\site-packages (from requests->-r requirements.txt (line 1)) (2.10)
Requirement already satisfied: soupsieve>1.2 in c:\python38\lib\site-packages (from beautifulsoup4->-r requirements.txt (line 2)) (2.0.1)
Installing collected packages: urllib3, requests, beautifulsoup4, win32-setctime, lxml, pysocks
Successfully installed beautifulsoup4-4.9.1 lxml-4.5.2 pysocks-1.7.1 requests-2.24.0 urllib3-1.25.10 win32-setctime-1.0.1

T:\#of2>py start_ofd.py
The .settings\config.json file has been created. Fill in whatever you need to fill in and then press enter when done.

Site: 0 = onlyfans | 1 = starsavn | 2 = fourchan | 3 = bbwchan | x = Exit
0
Unable to create session
There's nothing to scrape.
Site: 0 = onlyfans | 1 = starsavn | 2 = fourchan | 3 = bbwchan | x = Exit
x

T:\#of2>py start_ofd.py
Site: 0 = onlyfans | 1 = starsavn | 2 = fourchan | 3 = bbwchan | x = Exit
0
Unable to create session
There's nothing to scrape.
Site: 0 = onlyfans | 1 = starsavn | 2 = fourchan | 3 = bbwchan | x = Exit
x
```
If you need any more information, just ask (I'm fairly new to github and issues, so I didn't know how much to put in here)
Any help would be appreciated.

**Comment by orthoron** [comment]
This is my exact problem at the moment also. My thread was marked closed for some reason.

**Comment by DIGITALCRIMINAL** [comment]
My bad, fixed

**Comment by qazsedcftgbhujmkol** [comment]
I git cloned the repo a few minutes ago.
I installed requirements via pip.
I ran start_ofd.py
I filled in the auth_id, sess, user-agent and auth_uniq_ values.
I pressed enter (to proceed and have the script load the config)
I still get:
```
T:\#OnlyFans>py start_ofd.py
Site: 0 = onlyfans | 1 = starsavn | 2 = fourchan | 3 = bbwchan | x = Exit
0
Unable to create session
There's nothing to scrape.
Site: 0 = onlyfans | 1 = starsavn | 2 = fourchan | 3 = bbwchan | x = Exit
```

I ran the script using an IDE.
modules/onlyfans.py:613
`for proxy2 in proxy:` , my <proxy> variable is an empty list.

I hope that helps
